docs/abims.md

```diff
@@ -0,0 +1,46 @@
+# nf-core/configs: ABiMS Configuration
+
+All nf-core pipelines have been successfully configured for use on the ABiMS cluster.
+
+To use, run the pipeline with `-profile abims`. This will download and launch the [`abims.config`](../conf/abims.config) which has been pre-configured with a setup suitable for the ABiMS cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+## Request an account
```

Comment:
```suggestion
## Request an account

```

Reply:
Just need a blank line here to have a correct markdown linting

Reply:
Oups 👍